File: LEETCODE/trees/traversalTechniques/DeptFirstSearch.java/InorderTraversal.java
import LEETCODE.trees.BinaryTree;

public class InorderTraversal {

    public static void inorder(BinaryTree.Node root) {
        if( root == null){
            return ;
        }
        inorder(root.left);
        System.out.println(root.data);
        inorder(root.right);
    }

    public static void main(String[] args) {

        BinaryTree.Node root = BinaryTree.createTree();

        inorder(root);
    }
}

File: LEETCODE/trees/traversalTechniques/DeptFirstSearch.java/InorderTraversal.cpp
#include<bits/stdc++.h>
using namespace std;

struct Node{
    int data;
    Node* left;
    Node* right;
    Node(int data){
        this->data = data;
        this->left = nullptr;
        this->right = nullptr;
    }
};

int inorder(Node* root, vector<int> &ans){
    if(root == nullptr){
        return ;
    }
    inorder(root->left, ans);
    ans.push_back(root->data);
    inorder(root->right, ans);
}

int main(){    
    return 0;
}